src/jsx-parser.ts

```diff
@@ -256,9 +266,12 @@ export class JSXParser extends Parser {
         }
 
         this.scanner.throwUnexpectedToken();
+
+        /* istanbul ignore next */
+        throw new Error('Unreachable');
```

Comment:
@DanielRosenwasser Somehow it still does not work. I think it's because `this.scanner.throwUnexpectedToken()` does not directly throw, it calls `this.scanner.errorHandler.throwError(...)` which does throw.

Strangely, if I inline the content of `this.scanner.throwUnexpectedToken()` here, then the compiler is quite happy.